Snakemake/config/tsinfer_Eddie.yaml

```diff
@@ -6,6 +6,7 @@ vcfDir: "/home/v1jobste/jobsteter/HighlanderLab/share/Snakemake/Data"
 #combinedVCFs: null
 merge: "yes"
 bcftoolsModule: "igmm/apps/bcftools/1.9"
+vcftoolsModule: "igmm/apps/vcftools/0.1.13"
 ancestralAllele: "/home/v1jobste/jobsteter/HighlanderLab/share/Snakemake/Data/AncestralAlleleVcf.txt"
 meta: "/home/v1jobste/jobsteter/HighlanderLab/share/Snakemake/Data/SampleMetaData.csv"
```

Comment:
also, if you are now defining running requirements in the rules (time, mem etc) better to pass it in the config as well? Some steps can vary considerably from species to species.

Reply:
Yes, I am planning to add all of them to the config - that is just there atm to test